doc/tutorial/linear-algebra/tutorial-basic-linear-algebra.doc

```diff
@@ -0,0 +1,295 @@
+(10x10)x(10x10) - Naive code                            100           30     2.055 ms 
+                                                     667 ns       665 ns       671 ns 
+                                                       9 ns         1 ns        20 ns 
+                                                                                      
+(10x10)x(10x10) - ViSP                                  100           46    2.0654 ms 
+                                                     490 ns       461 ns       542 ns 
+                                                     194 ns       122 ns       278 ns 
+                                                                                      
+(20x20)x(20x20) - Naive code                            100            4    2.5972 ms 
+                                                   6.441 us     6.434 us     6.466 us 
+                                                      60 ns         8 ns       141 ns 
+                                                                                      
+(20x20)x(20x20) - ViSP                                  100           18     2.133 ms 
+                                                   1.179 us     1.163 us     1.253 us 
+                                                     149 ns         9 ns       355 ns 
+                                                                                      
+(6x200)x(200x6) - Naive code                            100            3    2.1447 ms 
+                                                   7.172 us     7.159 us     7.206 us 
+                                                      94 ns         5 ns       189 ns 
+                                                                                      
+(6x200)x(200x6) - ViSP                                  100           12    2.2092 ms 
+                                                   1.852 us     1.839 us     1.901 us 
+                                                     120 ns         2 ns       287 ns 
+\endverbatim
+
+\subsection linear-algebra-compar Lapack performance comparison
+
+The following table gives some results obtained on Ubuntu 18.04 with a DELL Laptop equipped with an Intel core i7-8650U CPU @ 1.90GHz that has 8 cores and 32 GB RAM. Results were obtained using:
+\verbatim
+$ ./perfMatrixMultiplication --benchmark --lapack-min-size 0
+\endverbatim
+that was build setting the appropriate `USE_BLAS/LAPACK=[ MKL | OpenBLAS | Netlib | Atlas | GSL ]` CMake option during configuration.
+
+Moreover, when `OpenBLAS`, `Netlib` or `Atlas` were selected we used `update-alternatives` tool to select the right Lapack library using:
+\verbatim
+$ sudo update-alternatives --config liblapack.so.3-x86_64-linux-gnu
+\endverbatim
+
+Next table summarises the results we obtained:
+
+| Operation Type |   Operation Size  |    MKL    |  OpenBLAS |   Atlas   |   Netlib  |     GSL    |  build-in  | Without Lapack |
```

Comment:
For me the timing with Netlib is not correct. It should be more or less similar to built-in.

I will try to run the different benchmark to check this.

Reply:
I don't know. For me `built-in` is just a conversion from Fortran to C. If I'm right it makes sense to have better results with `Netlib` than `built-in`